tm2/pkg/bft/privval/config.go

```diff
@@ -0,0 +1,108 @@
+package privval
+
+import (
+	"fmt"
+	"log/slog"
+	"path/filepath"
+
+	"github.com/gnolang/gno/tm2/pkg/bft/privval/signer/local"
+	rsclient "github.com/gnolang/gno/tm2/pkg/bft/privval/signer/remote/client"
+	"github.com/gnolang/gno/tm2/pkg/bft/types"
+	"github.com/gnolang/gno/tm2/pkg/crypto/ed25519"
+	"github.com/gnolang/gno/tm2/pkg/errors"
+)
+
+// PrivValidatorConfig defines the configuration for the PrivValidator, with a local or remote
+// signer, including network parameters and filepaths.
+type PrivValidatorConfig struct {
```

Comment:
Why not have 2 different configs, for different signer implementations (local / remote)?

Reply:
I not sure to understand what you mean: this is the `PrivValidator` config that encapsulates the `Signer` and the `State`. 
What change are you suggesting exactly?

Reply:
```go
type PrivValidatorConfig struct {
	// File path configuration.
	RootDir     string `json:"home" toml:"home"`
	SignState   string `json:"sign_state" toml:"sign_state" comment:"Path to the JSON file containing the last validator state to prevent double-signing"`
	LocalSigner string `json:"local_signer" toml:"local_signer" comment:"Path to the JSON file containing the private key to use for signing using a local signer"`

	// Remote Signer configuration.
	RemoteSigner *rsclient.RemoteSignerClientConfig `json:"remote_signer" toml:"remote_signer" comment:"Configuration for the remote signer client"`
}
```

Where do we use `RemoteSigner`?
It seems strange to have both `LocalSigner` and `RemoteSigner` in the `PrivValidatorConfig`, given they are either-or

Reply:
As discussed in DM, we will keep this configuration as is.

Reply:
As concluded in DM, we will keep the configuration as is.